### PR TITLE
📖 DOC: Update Readme

### PR DESCRIPTION
**Pull request by meerhamzadev**
✅ Add a banner image
✅ Add stat badges
✅ Add contributors section
✅ Beautify overall readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,23 @@
-# Module Federation Support for Node Environments
+![banner image](./assets/banner.png)
+<div align="center">
+	<!--  for version -->
+  <img src="https://img.shields.io/npm/v/<package-name>" alt="version" >
+	<img src="https://img.shields.io/apm/l/atomic-design-ui.svg?" alt="license" >
+  <!-- for downloads -->
+  <img src="https://img.shields.io/npm/dt/?<package-name>?color=%2317BCB8" alt="downloads">
```
Comment (ScriptedAlchemy) [inline README.md:7]: package name is wrong here

File: README.md [hunk 2]
```diff
@@ -1,8 +1,23 @@
-# Module Federation Support for Node Environments
+![banner image](./assets/banner.png)
+<div align="center">
+	<!--  for version -->
+  <img src="https://img.shields.io/npm/v/<package-name>" alt="version" >
```
Comment (ScriptedAlchemy) [inline README.md:4]: package name is wrong here

**Review by ScriptedAlchemy — CHANGES_REQUESTED** [review]
see comments

File: README.md [hunk 3]
```diff
@@ -1,8 +1,23 @@
...
+<!-- when you published it on npm simply replace <package-name> with the name of package  -->
 
-This package exposes two Webpack Plugins to bring the concept and power of Module Federation to NodeJS. This will allow your server to fetch chunks across the network allowing for distributed deployments of federated applications.
+<p align="center">
+<strong>A package to bring the concept and power of module federation to NodeJS.</strong>
+</p>
 
-## Installation
+## ⚡ Features
+- Exposes two Webpack Plugins to enable Module Federation.
```
Comment (ScriptedAlchemy) [inline README.md:16]: also check the recent merged, we added UniversalFederationPlugin which wraps both node federation and normal module federation into one plugin for client and server

File: README.md (hunk 2 above)
Comment (meerhamzadev) [reply README.md:4]: Actually, I didn't know the package name that's why I add it as a placeholder.

File: README.md (hunk 3 above)
Comment (meerhamzadev) [reply README.md:16]: ok, I will fix all this, tomorrow. Can you please mention the package name?

**Pull request closed without merging**